tasks/check-fbc-packages/check-fbc-packages.yaml

```diff
@@ -0,0 +1,55 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: check-fbc-packages
+  labels:
+    app.kubernetes.io/version: "1.4.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Task to create internalrequests to add contributions to index images
```

Comment:
maybe this was leftover from the copied task?

Reply:
It sure was! Fixing now.